lua/apg/modules/ghosting.lua

```diff
@@ -31,14 +31,10 @@ local mod = "ghosting"
 local ENT = FindMetaTable( "Entity" )
 APG.oSetColGroup = APG.oSetColGroup or ENT.SetCollisionGroup
 function ENT:SetCollisionGroup( group )
-	if APG.isBadEnt( self ) and APG.getOwner( self ) then
-		if group == COLLISION_GROUP_NONE then
-			if not self.APG_Frozen then
+	if APG.isBadEnt( self ) and APG.getOwner( self ) and group == COLLISION_GROUP_NONE and not self.APG_Frozen  then

```

Comment:
Another pedantic suggestion, but perhaps this could be reworked so the conditional has very little logic in it.

```suggestion
local isBadEnt = APG.isBadEnt( self )
local isValidOwner = APG.getOwner( self )
local groupIsNone = group == COLLISION_GROUP_NAME
local isNotFrozen = not self.APG_Frozen

local shouldMakeInteractable= isBadEnt and hasValidOwner and groupIsNone and isNotFrozen

if shouldMakeInteractable then
    group = COLLISION_GROUP_INTERACTIVE
end
```

I understand why this might look unnecessary, but my defense for suggesting this would be that it spread the logic out into multiple, easy-to-understand steps that walks the reader through what we're checking and why. It is more lines, but I think it might read better and be more maintainable in the future.

Up to you, just wanted to suggest it!